anaconda.spec.in

```diff
@@ -47,6 +47,11 @@ Source0: https://github.com/rhinstaller/%{name}/releases/download/%{name}-%{vers
 %define utillinuxver 2.15.1
 %define rpmostreever 2023.2
 
+# we generally want Fedora ELN packages to be like RHEL; special cases need to check %{fedora} explicitly
+%if "0%{?fedora}" == "eln"
+%define rhel %{fedora}
+%endif
+
```

Comment:
I am a bit confused about the purpose here. Isn't rhel macro already defined for ELN builds ?
https://docs.fedoraproject.org/en-US/eln/buildroot/#_distribution_related_macro_definitions

Reply:
I did not realize. thanks for this. In this case this PR is useless. Closing